Source/WebCore/css/parser/CSSTokenizerInputStream.cpp

```diff
@@ -42,18 +44,35 @@ CSSTokenizerInputStream::CSSTokenizerInputStream(const String& input)
 
 void CSSTokenizerInputStream::advanceUntilNonWhitespace()
 {
-    // Using ASCII whitespace here rather than CSS space since we don't do preprocessing
-    if (m_string->is8Bit()) {
-        auto characters = m_string->span8();
+    auto advance = [this](auto characters) {
         while (m_offset < m_stringLength && isASCIIWhitespace(characters[m_offset]))
             ++m_offset;
+    };
+
+    if (m_string->is8Bit()) {
+        advance(m_string->span8());
     } else {
-        auto characters = m_string->span16();
-        while (m_offset < m_stringLength && isASCIIWhitespace(characters[m_offset]))
-            ++m_offset;
+        advance(m_string->span16());
     }
```

Comment:
For WebKit coding style, should omit the braces here since these are one-line `if`/`else` blocks.